tools/pot/tests/test_sanity.py

```diff
@@ -31,6 +31,20 @@
                                                                               'accuracy@top5': 0.91},
      {}, 'CPU'),
 
+    ('resnet-50-tf', 'tf', 'AccuracyAwareQuantization', 'performance', 300, {'accuracy@top1': 0.760,
```

Comment:
The new tests increase pre-commit execution up to ~2x times (in comparison with the preview runs). Do we really need to check such a huge model like resnet-50 for both (GNA and GNA3) devices? cc @AlexKoff88

Reply:
I agree that we need to skip these unit test because of the CI running time.